pkg/metricsstore/metricsstore.go

```diff
@@ -158,6 +164,21 @@ func init() {
 			Help:      "Histogram to track time spent to issue xds certificate",
 		},
 		[]string{})
+
+	/*
+	 * Errcode metrics
+	 */
+	defaultMetricsStore.ErrcodeCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: metricsRootNamespace,
+			Name:      "errcode_count",
```

Comment:
Set the `Subsystem` to `error`.

Reply:
`osm.error.count` reads better than `osm.error.errcode_count`, given the labels are enough to carry additional context.